src/components/StatTile.vue

```diff
@@ -121,6 +131,39 @@ export default class StatTile extends Vue {
   // Expose RankConfig to template
   RankConfig = RankConfig;
 
+  get pluralismForPropertyType(): string {
+    let pluralismForProperty = "s";
```

Comment:
Why does this start as `"s"` if you completely set it fresh in every condition, including the `else`?